src/pharmpy/modeling/basic_models.py

```diff
@@ -124,6 +124,7 @@ def create_basic_pk_model(
         filename_extension='.mod',  # Should this really be needed?
         dataset=df,
         datainfo=di,
+        value_type='PREDICTION',
```

Comment:
This should be the default in `create` so no need to specify. If it isn't already the default that should be changed.